CHANGELOG.md

```diff
@@ -5,6 +5,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased][Unreleased]
+
+### Unreleased
+
+### Added
+
+- docs(readme): add support for AKS 1.22 [#2076][#2076]
+
+### Changed
+
+<!-- markdown-link-check-disable -->
+[Unreleased]: https://github.com/SumoLogic/sumologic-kubernetes-collection/compare/v2.4.1...release-v2.4
+<!-- markdown-link-check-enable -->
```

Comment:
You don't need those markdown link annotations since the tag is already there

```suggestion
[Unreleased]: https://github.com/SumoLogic/sumologic-kubernetes-collection/compare/v2.4.1...release-v2.4
```